docs/python/quickstart.md

```diff
@@ -5,4 +5,99 @@ sidenav: side-nav-python.html
 type: markdown
 ---
 
-## Abseil Quickstart (Python)
\ No newline at end of file
+# Abseil Quickstart (Python)
+
+This quick tutorial will get you up and running with Python Abseil. Python
+Abseil works with Python 2 and 3. Please see the
+[Programming Guides](/docs/python/guides) for details about specific modules.
+
+## Installation
+
+To install the package, simply run:
+
+```sh
+pip install absl-py
```

Comment:
Add a prerequisites section, with links:

* Python
* pip
* git

See http://abseil.io/docs/cpp/quickstart.html for examples.